lib/Tool/Text.php

```diff
@@ -176,7 +176,10 @@ public static function wysiwygText(?string $text, array $params = []): ?string
                         if (!empty($additionalAttributes)) {
                             $replacement .= ' ' . array_to_html_attribute_string($additionalAttributes);
                         }
-                        $newTag = preg_replace($pattern, $replacement, $oldTag);
+
+                        $newTag = self::addHrefAttributeToTag($oldTag);
```

Comment:
We need to consider `src=""` attribute from `<img` tags here as well, so probably we need to make the code a bit more generic 😊

We should also explain why we're doing this here, because there's no obvious reason for doing this 😉